Automattic/wp-calypso
# Sites Dashboard: Persist search value as `?search=` and between tabs

Fixes https://github.com/Automattic/wp-calypso/issues/65605
Fixes https://github.com/Automattic/wp-calypso/issues/65603

## Proposed Changes

Persists the search input value in the URL as `?search=<value>` and also when switching between tabs.

<img width="932" alt="image" src="https://user-images.githubusercontent.com/36432/179278666-e479d7f2-cf15-49f7-869f-b8882ce828ff.png">

## Testing Instructions

1. Navigate to `/sites-dashboard`.
2. Enter a search query.
3. See search query appended to URL as `?search=<value>`.
4. Switch to a different launch status tab.
5. See the search query appear in the search field, and applied to the list of sites.
6. Clear the search field.
7. See the search query removed from the URL.

## Pre-merge Checklist

Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
